spec/indent/method_definitions_spec.rb

```diff
@@ -74,6 +74,51 @@ def foo(
         # Reference: https://github.com/vim-ruby/vim-ruby/issues/450
         def self.foo = puts(bar)
         def bar.foo = puts(baz)
+
+        def +(other) = to_r + other
+        def -(other) = to_r - other
+        def *(other) = to_r * other
+        def **(other) = to_r ** other
+        def /(other) = to_r / other
+        def %(other) = to_i % other
+        def &(other) = to_i & other
+        def ^(other) = to_i ^ other
+        def >>(other) = to_i >> other
+        def <<(other) = to_i << other
+        def ==(other) = to_r == other
+        def !=(other) = to_r != other
+
+        def ===(other) = to_r === other # didn't need a change, but also broken
+        # def foo===(other) = fixme
```

Comment:
The test seems to fail for me at this point and continue to fail down the line, does it pass for you? If so, there might be some local setting that needs to be ignored in the regex.

Reply:
Sorry I think I just hadn't finished the work 😬 

The comments meant that I didn't need to change anything for these to work, but apparently after rebasing I did need to handle these cases specifically in the regex.

fixme meant they didn't work, but I think I was just missing the `.` to define singleton methods instead.